event hubs migration guide.md

```diff
@@ -0,0 +1,81 @@
+# Migration Guide (EventHubs 2 to 5)
+
+## Class changes
+
+`EventHubClient` has been split into two clients - `EventHubProducerClient` (for sending
+messages) and `EventHubConsumerClient` (for subscribing to messages).
+
+| In v2                                       | Equivalent in v5                                                 | Sample |
+|---------------------------------------------|------------------------------------------------------------------|--------|
+| `EventHubClient.createFromConnectionString` | `new EventHubProducerClient()` or `new EventHubConsumerClient()` | [All](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/)
```

Comment:
What about when using AAD?

`EventHubClient.createFromAADTokenCredentials`  would become
`new EventHubProducerClient()` and `new EventHubConsumerClient()` using any of the [TokenCredentials](https://www.npmjs.com/package/@azure/identity)

Reply:
Awesome, will add.